histogram/src/sparse.rs

```diff
@@ -124,22 +124,20 @@ impl SparseHistogram {
         })
     }
 
-    /// Returns a new histogram with a reduced grouping power. The specified
-    /// reduction factor should be 0 < factor < existing grouping power.
+    /// Returns a new histogram with a reduced grouping power. The reduced
+    /// grouping power should lie in the range (0, existing grouping power).
```

Comment:
I'd prefer to use rust range syntax here if possible. Acceptable values do include 0 but not the current grouping power. It's unclear with it written the way it is right now.